Framework/DataHandling/src/LoadNexusProcessed.cpp

```diff
@@ -373,28 +374,28 @@ Workspace_sptr LoadNexusProcessed::doAccelleratedMultiPeriodLoading(NXRoot &root
 void LoadNexusProcessed::execLoader() {
 
   API::Workspace_sptr tempWS;
-  int nWorkspaceEntries = 0;
+  size_t nWorkspaceEntries = 0;
```

Comment:
Not a problem in this case, but openmp on windows used to not work with unsigned integer types. I keep meaning to check if they are supported yet. That is a frequent cause of casting integer types in mantid.